src/main/java/bio/terra/cli/command/helperclasses/options/NotebookName.java

```diff
@@ -0,0 +1,65 @@
+package bio.terra.cli.command.helperclasses.options;
+
+import bio.terra.cli.command.exception.UserActionableException;
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+import bio.terra.cli.service.WorkspaceManager;
+import bio.terra.cloudres.google.notebooks.InstanceName;
+import bio.terra.workspace.model.GcpAiNotebookInstanceAttributes;
+import bio.terra.workspace.model.ResourceDescription;
+import picocli.CommandLine;
+
+/**
+ * Command helper class for identifying a notebook by either the resource name of the GCP instance
+ * name in `terra notebooks` commands.
```

Comment:
typo-- or instance id?

Reply:
`or` (not `of`, thanks) instance name really, cause the GCP instance id isn't enough to specify the notebook, it needs the full "GCP instance name" https://cloud.google.com/ai-platform/notebooks/docs/managed/reference/rest/v1beta1/projects.locations.instances